bindings/python/libs/client/file.py

```diff
@@ -42,7 +43,7 @@ def __exit__(self, type, value, traceback):
   def __iter__(self):
     return self
 
-  def next(self):
+  def __next__(self):
```

Comment:
In order to maintain compatibility with python 2 you need to add `next = __next__` to the class.